content/post/practical_ml.md

```diff
@@ -0,0 +1,77 @@
+---
+author: Ilan Man
+title: "Practical tips for real-world Data Science"
+publishDate: "2018-05-15"
+draft: true
+tags: 
+ - datascience
+---
+
+Many Data Scientists today come from a hard science background - statistics, math, physics. Hard sciences have a bias towards empirical and objective truths: a correct answer exists and we can find it by employing the scientific method, usually manifested by a formulaic approach. While not a controversial statement in itself, many years of studying and application of such a mantra can collide with the realities of the Business world. There, it becomes increasingly difficult to perfectly apply the theory. As a result, the practitioner should understand how to adjust their model and their approach accordingly.
+
+In this post we'll discuss a few considerations that the Data Scientist should note during the model building and deployment process. As a running example to help illustrate this:
```

Comment:
Consider moving the running example below the fold

Reply:
Sounds good.